platform/src/main/java/org/stellar/anchor/platform/payment/observer/stellar/StellarPaymentObserver.java

```diff
@@ -64,6 +68,16 @@ public void start() {
   /** Graceful shutdown. */
   public void shutdown() {
     this.stream.close();
+    this.stream = null;
+  }
+
+  private void restart() {
+    if (this.stream != null) {
```

Comment:
Can we log restart here?

Reply:
Updated in https://github.com/stellar/java-stellar-anchor-sdk/pull/607/commits/247705aab4bb5e0d048fba7d6ae2d9c768103249.